modules/arm_plugin/armplg_build.sh

```diff
@@ -92,8 +92,9 @@ PYTHONVER=`ls /usr/include | grep "python3[^m]*$"` && \
 cmake -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DBUILD_LIST=imgcodecs,videoio,highgui,gapi,python3 \
       -DBUILD_opencv_python2=OFF -DBUILD_opencv_python3=ON -DOPENCV_SKIP_PYTHON_LOADER=ON \
       -DPYTHON3_LIMITED_API=ON -DPYTHON3_PACKAGES_PATH=$STAGING_DIR/opencv/python \
-      -DPYTHON3_INCLUDE_PATH=/usr/include/${PYTHONVER}m \
```

Comment:
later in this file we also have `m` suffix, looks unaligned now

Reply:
Removed `m` suffix from the library name